machine/dni/dni_3048up/INSTALL

```diff
@@ -0,0 +1,92 @@
+============================================================
+Installing ONIE on DNI 3048UP
+============================================================
+
+Cross-Compiling ONIE
+====================
+
+Change directories to ``build-config`` to compile ONIE.
+
+To compile ONIE first change directories to ``build-config`` and then
+type ``"make MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all"``.
+For example::
+
+  $ cd build-config
+  $ make -j4 MACHINEROOT=../machine/dni/ MACHINE=dni_3048up all
+
+When complete, the ONIE binaries are located in
+``build/images``::
+
+  -rw-r--r-- 3604480 May  7 11:38 onie-dni_3048up-r1.bin
```

Comment:
As per  line 35,  u-boot and ONIE kernel are installed separately, seems to be inline with nor flash layour  dni_3048up.dts where u-boot+env and onie binaries need to be generated separately.

Please fix onie-rom.conf format as uboot_ubootenv-up and fix install instructions to reflect that.

Please check qemu_armv7a INSTALL instructions which has similar layout to this board.
https://github.com/opencomputeproject/onie/blob/master/machine/qemu_armv7a/INSTALL